src/data/scripts/building-owners/find_cha_buildings.py

```diff
@@ -18,6 +28,14 @@
                     found.append((whole_file_line["ID"], whole_file_line["Property Name"]))
                     break
 
+
+print(f"Found {len(found)} CHA Addresses in Benchmarking Data Using \"{cha_buildings_filename}\"!")
```

Comment:
I made all the building owner files output in the same way to make them more consistent.